src/main/java/step2/inputView/InputView.java

```diff
@@ -0,0 +1,44 @@
+package step2.inputView;
+
+import step2.domain.game.GameData;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Scanner;
+import java.util.stream.Collectors;
+
+public class InputView {
+    private static final Scanner sc = new Scanner(System.in);
```

Comment:
상수이므로 대문자로 선언 되어야 합니다 😄 
```suggestion
    private static final Scanner SC = new Scanner(System.in);
```